codespan/src/filemap.rs

```diff
@@ -117,6 +145,22 @@ impl FileMap {
         self.span
     }
 
+    pub fn offset(&self, line: LineIndex, column: ColumnIndex) -> Option<ByteOffset> {
+        self.line_offset(line).ok().and_then(|mut offset| {
+            offset += ByteOffset::from(column.0 as i64);
```

Comment:
Should we have an error if this runs into the next line?

Reply:
Kind of weird it is not called `byte_offset`, but then it would be inconsistent with `line_offset` 🤔

Reply:
> Should we have an error if this runs into the next line?

Probably should have an error for that and to capture the error from `line_offset` as well. It gets a bit verbose when having precise errors.